lib/trento/domain/cluster/commands/deregister_cluster_host.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Trento.Domain.Commands.DeregisterClusterHost do
+  @moduledoc """
+    Deregister a cluster host
+  """
+  @required_fields :all
+
+  use Trento.Command
+
+  defcommand do
+    field :host_id, Ecto.UUID
+    field :cluster_id, Ecto.UUID
```

Comment:
Shouldn't we have here `deregistered_at` field?
At the end we will cascade this value from events to commands in the process manager

Reply:
+1